lib/Doctrine/DBAL/Exception/InvalidArgumentException.php

```diff
@@ -37,4 +37,9 @@ public static function fromEmptyCriteria()
     {
         return new self('Empty criteria was used, expected non-empty criteria');
     }
+
+    public static function fromUnsupportedOption(string $name) : self
+    {
+        return new self(sprintf('The "%s" option is not supported', $name));
+    }
```

Comment:
Unused at the moment, should be used in 3.0.

Reply:
Should I remove it or should I strive to make the diff of the 3.0 PR minimal?

Reply:
I'd remove it since it's not used anywhere.